docs/collaboration.md

```diff
@@ -0,0 +1,105 @@
+
+An example of a Deployment file is
+`/.posit/publish/deployments/deployment-58A3.toml`.
+
+Utilizing both of these files, alongside a credential and collaborator access
+on Posit Connect, you and your collaborators can update and configure a single
+piece of content.
+
+## Collaborator access
+
+The original creator of a piece of content on Posit Connect needs to give others
+collaborator access to update the content.
+
+See the [Posit Connect User Guide - Access list settings](https://docs.posit.co/connect/user/content-settings/#set-collaborators)
+documentation for more information on giving users access to update a piece of
+content.
+
+## Source control setup
+
+Deployment and Configuration files found in `./posit/publish` are intended
+to be committed into source repositories.
+
+Once tracked in source control, anyone with access can easily collaborate on the
+same piece of content.
+
+After a deploy be sure to commit any changes to the Deployment and Configuration
+files. Keeping them in sync with the content on Posit Connect will ensure that
+collaborators have the most up-to-date details to continue where others left
+off.
+
+## Content bundle setup
+
+If you and your collaborators aren't using source control, a content bundle
+containing the source code and data for a piece of content can be downloaded
+from Posit Connect. Content bundles are created when content is published and
+they work great with Posit Publisher.
+
+When deploying, include the Deployment and Configuration files so others can
+use them to collaborate after they download the content bundle.
+
+Posit Publisher includes both files automatically in configurations it generates
+to make this effortless.
+
+See the
+[Posit Connect User Guide - Source Versions](https://docs.posit.co/connect/user/source-versions/)
+documentation for more information on the content bundle source versions and
+how to download them.
+
+> [!NOTE]
+> The content bundle will only contain files sent to Posit Connect. Be sure any
+> files that you would like collaborators to have access to are in the
+> Configuration's included `files`.
+> For more details see the [Project Files documentation for Posit Publisher](./vscode.md#project-files).
+
+## Tips
+
+### Stay updated while collaborating
+
+While collaborating on a piece of content, stay updated with the latest changes
+others have made.
+
+To do so, either pull down the latest changes from the repository or download
+the content bundle from the latest Source Version on Posit Conect.
+
+Doing so ensures that you have the most up-to-date content, but also
+configuration settings and deployment details.
+
+### First time deploying
+
+If a deployment is pointed to a server that you have not deployed to before,
+you may need to setup your credentials.
+
+Posit Publisher makes this easy by prompting you to _Create a new Credential_
+when the selected deployment requires a credential you don't have.
+
+TODO: SCREEN SHOT
```

Comment:
This is covered in our general publishing docs, yeah? If yeah I think we can 🚮

Reply:
We don't mention this functionality in the [VS Code doc in the repo](https://github.com/posit-dev/publisher/blob/main/docs/vscode.md#credentials) nor in [the Connect User Guide doc](https://docs.posit.co/connect/user/publishing-positron-vscode/). I can move it into that VS Code document under Credentials though.

Reply:
Moved this over to the VS Code doc since it was more generally applicable.